integrationtest/src/main/scala/io/gearpump/integrationtest/minicluster/MiniCluster.scala

```diff
@@ -125,6 +125,10 @@ class MiniCluster(
     }
   }
 
+  def getMasters = {
```

Comment:
The `getMasterHosts` method only returns the list of masters' hosts, I also need the ports information of masters to be verified.